package.json

```diff
@@ -4,21 +4,19 @@
   "workspaces": [
     "packages/*"
   ],
+  "browser": "./examples/build.dev.js",
+  "browserMin": "./examples/build.prod.js",
```

Comment:
What are these two for? I thought that `browser` was a field for including a browserified version of something. But this package is private, so why do we need it?

Reply:
These two are [imported and used](https://github.com/schneidmaster/slate/blob/rollup/rollup.config.js#L10) by the top-level `rollup.config.js` when building the examples. (I could hardcode it inside the rollup config, I was just keeping a consistent pattern with all the sub packages.)

Reply:
I think the `browser` has special meaning as a `package.json` field? I believe it's used in Browserify and others as the browser-preferred variant for packages. We might want to call it `umd` and `umdMin` instead I think to avoid this breaking stuff?

Reply:
Well in the child packages I think `"browser"` is appropriate (because the fields do point to the UMD/browser-preferred variant). And I don't think it matters a ton in the top-level package because it's private, so I opted for consistency with the child packages. I don't really care though, I can change it if you want 🙂

Reply:
I think `browser` isn't supposed to be UMD though, I think it is usually just an alternate, browser-compatible CommonJS version of the code.

Reply:
Oh fair enough, I guess I was under the wrong impression of what `browser` is for